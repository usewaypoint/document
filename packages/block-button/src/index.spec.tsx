import React from 'react';

import { render } from '@testing-library/react';

import { Button } from '.';

describe('block-button', () => {
  it('renders with default values', () => {
    expect(render(<Button />).asFragment()).toMatchSnapshot();
  });
});
