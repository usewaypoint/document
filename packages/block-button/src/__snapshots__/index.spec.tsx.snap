// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`block-button renders with default values 1`] = `
<DocumentFragment>
  <div>
    <a
      href=""
      style="color: rgb(255, 255, 255); font-size: 16px; font-weight: bold; background-color: rgb(153, 153, 153); border-radius: 4px; display: inline-block; padding: 12px 20px; text-decoration: none;"
      target="_blank"
    >
      <span>
        <!--[if mso]&gt;&lt;i style="letter-spacing: 20px;mso-font-width:-100%;mso-text-raise:30" hidden&gt;&nbsp;&lt;/i&gt;&lt;![endif]-->
      </span>
      <span />
      <span>
        <!--[if mso]&gt;&lt;i style="letter-spacing: 20px;mso-font-width:-100%" hidden&gt;&nbsp;&lt;/i&gt;&lt;![endif]-->
      </span>
    </a>
  </div>
</DocumentFragment>
`;
